src/iiiftilesource.js

```diff
@@ -181,20 +182,34 @@ $.extend( $.IIIFTileSource.prototype, $.TileSource.prototype, /** @lends OpenSea
     },
 
     /**
-     * Return the tileSize for the given level.
+     * Return the tileWidth for the given level.
      * @function
      * @param {Number} level
-    */
+     */
+    getTileWidth: function( level ) {
```

Comment:
Sorry ... I don't understand why there's no changes to ll 55-99 to actually get the tile width/height information.  It currently still uses the previous logic of width == height, unless width isn't even present.  So currently :-1: due to incompleteness.

I can build a demo IIIF server that supports non-square tiles for testing, if that would be valuable.